_users-and-settings/hosting.markdown

```diff
@@ -13,6 +13,32 @@ The repository you use must already exist. To make things easier, we suggest tha
 
 For help with GitHub Pages, check out [their help page](https://help.github.com/categories/github-pages-basics/).
 
+### Configuring GitHub Pages
+
+There are a few pieces of site metadata that you can override:
+
+{% highlight yaml %}
+github: [metadata]
+kramdown:
+  input: GFM
+  hard_wrap: false
+gems:
+  - jekyll-coffeescript
+  - jekyll-paginate
+{% endhighlight %}
+
+There are also a few pieces that you _can_ override:
```

Comment:
Is one of these lines supposed to say "can't"? At the moment they're both saying you _can_ override. Not sure what the difference is.